demo/clusters/openshift/add-certified-catalog-source.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Should we link these scripts in the README?

Reply:
I'm not sure. Actually I wasn't sure if the scripts were needed at all. Let's leave it as it is, anybody trying OpenShift will notice them, but it's important that they understand what the scripts  are doing and why.